config.rb

```diff
@@ -105,4 +104,8 @@
   def article_author(article)
     data.authors[article.data.author]['name']
   end
+
+  def get_full_post_url
```

Comment:
maybe `full_article_url` to keep it consistent with the method above?

and i'd pass the `article` as parameter.